Dockerfile

```diff
@@ -1,7 +1,20 @@
-FROM scratch
-COPY ./cacert.pem /etc/ssl/certs/ca-certificates.crt
-COPY cardigann-linux-amd64 /cardigann
+FROM alpine:latest
+
+ENV CARDIGANN_PKG cardigann-stable-linux-amd64.tgz
+ENV CONFIG_DIR /etc/cardigann
+
+ADD https://bin.equinox.io/c/3u8U4iwUn6o/${CARDIGANN_PKG} /
```

Comment:
I think both use-cases have merit, however I also agree that adding it from the local filesystem is more flexible, I'll revert that.